Components/Screens/UserHomeScreen.js

```diff
@@ -7,6 +7,14 @@ import { PropTypes } from 'prop-types';
 import { styles } from './styleUserHomeScreen';
 
 export class UserHomeScreen extends Component {
+	componentWillUnmount(){
+		console.log('unmounting')
+	}
+
+	componentDidMount(){
+		console.log('mounting')
+	}
+
```

Comment:
Do we still need these in here? I thought these just show up in the console.

Reply:
No, thank you for catching that.